crates/turbopack-ecmascript/js/src/runtime.nodejs.js

```diff
@@ -0,0 +1,25 @@
+/** @typedef {import('../types/backend').RuntimeBackend} RuntimeBackend */
+
+/** @type {RuntimeBackend} */
+const BACKEND = {
+  loadChunk(chunkPath, from, onLoad, onError) {
```

Comment:
Instead of `onLoad` and `onError` this can return a Promise instead

Reply:
I'll make that change. However, the semantics are slightly different: when loading a JS chunk, `resolve` will never actually be called inside of the `BACKEND`: it will instead be called when the evaluated chunk registers itself.